README.md

```diff
@@ -31,14 +35,7 @@ Before getting started, make sure you have the following prerequisites installed
 
 ## Setting up
 
-Start by cloning this repository to your system:
-
-```
-cd ~
-git clone https://github.com/IBM/ToDoBackend.git
-```
-
-Then, clone the testsuite you'll be using to verify your REST API:
+Start by cloning this repository to your system, this is the testsuite you'll be using to verify your REST API:
```

Comment:
"this repository" could perhaps be misinterpreted as the workshop repository? Maybe something like "Start by cloning the testsuite to your system. You'll be using this to verify your REST API:" WDYT?